js/controllers/authors.js

```diff
@@ -0,0 +1,69 @@
+/* global app */
+/* global angular */
+
+app.controller('authorsCtrl', ['$scope', 'authorService', 'enableEditService', 'bookService', function($scope, authorService, enableEditService, bookService, $index) {
+  
+  $scope.include = function(obj) {
+     authorService.updateAuthors(obj);
+   };
+   
+  $scope.temp = authorService.authorsList;
```

Comment:
temp is not a meaningful object name - please call it something else

Reply:
right, changed to authorData